sendgrid/cmd/send.go

```diff
@@ -0,0 +1,95 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/sendgrid/sendgrid-go"
+	"github.com/sendgrid/sendgrid-go/helpers/mail"
+)
+
+func Send(ctx context.Context, to, subject, textBody, htmlBody string) (string, error) {
+	// Retrieve and trim environment variables
+	sendGridAPIKey := strings.TrimSpace(os.Getenv("SENDGRID_API_KEY"))
+	if sendGridAPIKey == "" {
+		return "", fmt.Errorf("SENDGRID_API_KEY is not set")
+	}
+
+	fromName := strings.TrimSpace(os.Getenv("SENDGRID_FROM_NAME"))
+	if fromName == "" {
+		return "", fmt.Errorf("SENDGRID_FROM_NAME is not set")
+	}
+
+	fromEmail := strings.TrimSpace(os.Getenv("SENDGRID_FROM_EMAIL"))
+	if fromEmail == "" {
+		return "", fmt.Errorf("SENDGRID_FROM_EMAIL is not set")
+	}
```

Comment:
Are these supposed to come from a credential tool? There is no credential tool referenced in the `tool.gpt` file.

Reply:
Yeah, this is going to be updated shortly

Reply:
Updated to use credentials tool.

Reply:
Addressed now